Makefile

```diff
@@ -0,0 +1,59 @@
+##@ Makefile
+
+##@ Define the default version
+VERSION ?= latest
+
+tools.airbyte-ci.install: ## Install airbyte-ci
+	##@ Determine the operating system and download the binary
+	@OS=$$(uname); \
+	if [ "$$OS" = "Linux" ]; then \
+		URL="https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/airbyte-ci/releases/ubuntu/${VERSION}/airbyte-ci"; \
```

Comment:
Can we use `connectors.airbyte.com` URLs?

Reply:
GREAT CALL OUT! yes we can